main.cpp

```diff
@@ -651,6 +847,24 @@ void Motorbike::crash(int injuriesSustained)
     }
 }
 
+void Motorbike::buy(int bikesPurchased)
+{
+    std::string garage[5];
```

Comment:
Containers haven't been covered yet.
regardless, raw C-style containers are bad.
use `std::array` if you know the quantity or `std::vector<>` if you do not.
- [ ] revise